wp-content/themes/access/views/locations/single-location.twig

```diff
@@ -158,5 +158,5 @@
 
 {% block scripts_footer %}
 {{ function('enqueue_script', 'assets/js/main') }}
-{{ function('enqueue_script', 'assets/js/locations') }}
+{# {{ function('enqueue_script', 'assets/js/locations') }} #}
```

Comment:
Single locations are not using this script at the moment so commenting it out so it doesn't get requested.